otterdog/eclipse-openbsw.jsonnet

```diff
@@ -9,4 +9,15 @@ orgs.newOrg('eclipse-openbsw') {
       actions_can_approve_pull_request_reviews: false,
     },
   },
+  _repositories+:: [
+    orgs.newRepo('openbsw') {
+      private: true,
```

Comment:
is this intentional? this would create a private repo.

Reply:
yes...intentional until we have uploaded all code and have a valid gh-pages documentation page in place

Reply:
then add `allow_forking: false` as property or `members_can_fork_private_repositories: true` to the org settings if this repo should be forkable.

Reply:
tbh, its the first time I see that people are willing to create a private repo. Its certainly not needed imho.